### PR TITLE
Update requirements to work with Python>=3.10

### PR DESCRIPTION
**Pull request by amerikrainian**
Fixed the deprecated collections import in sound_lib

### CODE REVIEW

File: requirements-mac.txt [hunk 1]
```diff
@@ -2,4 +2,4 @@ wxpython
 pywin32
 pygame
 pyobjc
-pyinstaller==3.6
+pyinstaller==5.13
```
Comment (yncat) [inline requirements-mac.txt]: Would you mind reverting this specific change?
Probably PyInstaller on Windows works, but I remember the latest pyInstaller did not work on Mac, causing an app bundle error for some unknown reason. Also needs testing for the latest OSX version.

File: requirements-win.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
 wxpython
 pywin32
 pygame
+winpaths
```
Comment (yncat) [inline requirements-win.txt:4]: This was what I totally forgot to add. Thanks.

**Review by yncat — CHANGES_REQUESTED** [review]
Only one thing!

**Comment by amerikrainian** [comment]
Done.

**Review by yncat — APPROVED** [review]
LGTM!

**Pull request merged**